tests/policy.bats

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bats
+
+setup() {
+    load ../helpers/helpers.sh
+    wait_pods
+}
+
+teardown_file() {
+    load ../helpers/helpers.sh
+    kubectl delete pods --all
+    kubectl delete admissionpolicies,clusteradmissionpolicies --all -A
```

Comment:
Revert to helm values from initial installation, you didn't create any extra policies manually.

```suggestion
    helmer reset kubewarden-defaults
```